org.eclipse.winery.topologymodeler.ui/src/app/canvas/entities-modal/entities-modal.service.ts

```diff
@@ -44,8 +46,8 @@ export class EntitiesModalService {
         } else {
             URL = backendBaseURL + '/admin/namespaces/';
         }
-        return this.http.get(URL, this.options)
-            .map(res => res.json());
+        return this.http.get(URL, this.options).pipe(
```

Comment:
Please refactor the `Http` to the `HttpClient`

Reply:
😭